src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/RawAtlasPointAndLineSlicer.java

```diff
@@ -92,6 +92,28 @@ public Atlas slice()
         return atlasWithSlicedWaysAndPoints;
     }
 
+    /**
+     * Takes a location, a list of points from the raw Atlas at that location, and a list of point
+     * IDs. Ensures the tags for the point are updated, ensures the points are not removed from the
+     * raw Atlas, and adds the points the list of points for the line.
+     *
+     * @param location
```

Comment:
I believe those need descriptions too. I thought Checkstyle caught this... will have to look into that.

Reply:
fixed!